doc/nrf/drivers.rst

```diff
@@ -16,3 +16,4 @@ Here you can find documentation for drivers in the |NCS|, including API document
    :glob:
 
    drivers/*
+   drivers/*/*
```

Comment:
`**` is better: See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#id2

Reply:
Done